airflow/dags/sync_kernel_to_website.py

```diff
@@ -356,7 +357,10 @@ def register_journals(ds, **kwargs):
         t_journal = JournalFactory(resp_json)
         t_journal.save()
 
-        known_issues[get_id(journal.get("id"))] = resp_json.get("items", [])
+        known_issues[get_id(journal.get("id"))] = {
```

Comment:
mudar o nome da variável `known_issues` para `known_docs_bundles`.

Reply:
@robertatakenaka

Não sei se devemos trazer a nomeclatura do Kernel para o opac-airflow, se formos puristas os dois não estão nomeando corretamente!

Reply:
Não seria por ser nomenclatura do Kernel mas um nome que represente um conjunto de documentos, independente se é fascículo regular ou Ahead of prints.

Reply:
@jamilatta não se trata de nomenclatura de kernel.

Em `known_issues` contém o que?
a) contém somente fascículo? Então o nome está correto. 
b) contém fascículos + AOP? Então não deve se chamar `known_issues` e, sim, `known_bundles`

Reply:
@robertatakenaka 

Discordo de adicionar a nomeclatura **bundle** em qualquer dos casos, aqui não se trata de maço, acredito que estamos enviesados com as nomeclaturas utilizadas no Kernel para atribuir nomes apenas para uma estrutura de dados, discordo que termos uma chave no dicionário com 'aop' simboliza um **bundle**.

Temos somente um estrutura de dados de dicionários de dicionários, portanto, concordo em mantermos o nome **known_issues**!